{
 "PRed": [
  {
   "PR": {
    "number": 1,
    "state": "closed"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "python310"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  }
 ],
 "bad": false,
 "pinning_version": "2022.07.08.02.15.19",
 "pre_pr_migrator_attempts": {
  "python311": 827,
  "python312": 314
 },
 "pre_pr_migrator_status": {
  "python311": "not solvable (<a href=\"https://github.com/regro/cf-scripts/actions/runs/6289444872\">bot CI job</a>): main: ['osx_arm64_python3.8.____cpython: Encountered problems while solving:\\n  - nothing provides python >=3.12.0rc3,<3.13.0a0 needed by pyobjc-framework-quartz-9.2-py312h81bd7bf_1\\n']",
  "python312": "not solvable (<a href='https://github.com/regro/cf-scripts/actions/runs/8493990321'>bot CI job</a>) @ main\n<details>\n<div align=\"left\">\n<pre>\nRuntimeError('Download error (23) Failed writing received data to disk/application [https://conda.anaconda.org/msys2/noarch/repodata.json.zst]\\nFailure writing output to destination, passed 114 returned 115')\n</pre>\n</div>\n</details>"
 },
 "smithy_version": "3.21.0"
}